HUD/SetWarningMessage.md

```diff
@@ -5,87 +5,71 @@ ns: HUD
 
 ```c
 // 0x7B1776B3B53F8D74 0xBE699BDE
-void SET_WARNING_MESSAGE(char* entryLine1, int instructionalKey, char* entryLine2, BOOL p3, int p4, cs_type(AnyPtr) char* background, cs_type(AnyPtr) char* p6, BOOL showBg, int errorCode);
+void SET_WARNING_MESSAGE(char* line1TextLabel, int buttonsBitField, char* line2TextLabel, BOOL addNumber, int numberToAdd, cs_type(AnyPtr) char* substring1TextLabel, cs_type(AnyPtr) char* substring2TextLabel, BOOL showBg, int errorCode);
 ```
 
-**instructionalKey enum list**:
+Displays a warning screen for one frame.
 
-```
-enum INSTRUCTIONAL_BUTTON_TYPES
-{
-    NONE = 0,
-    SELECT = 1,
-    OK = 2,
-    YES = 4,
-    BACK = 8,
-    BACK_SELECT = 9,
-    BACK_OK = 10,
-    BACK_YES = 12,
-    CANCEL = 16,
-    CANCEL_SELECT = 17,
-    CANCEL_OK = 18,
-    CANCEL_YES = 20,
-    NO = 32,
-    NO_SELECT = 33,
-    NO_OK = 34,
-    YES_NO = 36,
-    RETRY = 64,
-    RETRY_SELECT = 65,
-    RETRY_OK = 66,
-    RETRY_YES = 68,
-    RETRY_BACK = 72,
-    RETRY_BACK_SELECT = 73,
-    RETRY_BACK_OK = 74,
-    RETRY_BACK_YES = 76,
-    RETRY_CANCEL = 80,
-    RETRY_CANCEL_SELECT = 81,
-    RETRY_CANCEL_OK = 82,
-    RETRY_CANCEL_YES = 84,
-    SKIP = 256,
-    SKIP_SELECT = 257,
-    SKIP_OK = 258,
-    SKIP_YES = 260,
-    SKIP_BACK = 264,
-    SKIP_BACK_SELECT = 265,
-    SKIP_BACK_OK = 266,
-    SKIP_BACK_YES = 268,
-    SKIP_CANCEL = 272,
-    SKIP_CANCEL_SELECT = 273,
-    SKIP_CANCEL_OK = 274,
-    SKIP_CANCEL_YES = 276,
-    CONTINUE = 16384,
-    BACK_CONTINUE = 16392,
-    CANCEL_CONTINUE = 16400,
-    LOADING_SPINNER = 134217728,
-    SELECT_LOADING_SPINNER = 134217729,
-    OK_LOADING_SPINNER = 134217730,
-    YES_LOADING_SPINNER = 134217732,
-    BACK_LOADING_SPINNER = 134217736,
-    BACK_SELECT_LOADING_SPINNER = 134217737,
-    BACK_OK_LOADING_SPINNER = 134217738,
-    BACK_YES_LOADING_SPINNER = 134217740,
-    CANCEL_LOADING_SPINNER = 134217744,
-    CANCEL_SELECT_LOADING_SPINNER = 134217745,
-    CANCEL_OK_LOADING_SPINNER = 134217746,
-    CANCEL_YES_LOADING_SPINNER = 134217748
-}
-```
+Available buttons for `buttonsBitField` (Can be added with `(1 << button1Index | 1 << button2Index | 1 << button3Index...)`):
+See the [`Controls Game Reference`](https://docs.fivem.net/docs/game-references/controls/) for information about Button Inputs.
 
-Note: this list is definitely NOT complete, but these are the ones I've been able to find before giving up because it's such a boring thing to look for 'good' combinations.
+You can find a list of most commonly used values [Here](https://gist.github.com/freedy69/a407b5d48225f1b9cff4d3e00216005a).
```

Comment:
While it's appreciated that you're hosting something for the docs, could you please put this list on the game docs and share the link here instead? It's always better FiveM is self-hosting.

Reply:
Sure but I'll remove this from this native doc for now